### ISSUE TITLE
Magic Resistance

### ISSUE THREAD

**Issue by ouroboroz1** [RetroWoW/PTR#84]
**Description**: Schools of Magic Resistance (Nature, Shadow, Fire, Frost, Holy, Arcane) not working as intended.

**Current behavior**:  Chance to resist or partially resist non-binary (haven't checked binary) spells differs from various sources. No change in chance to resist or partially resist noticeable at different resistances. Unable to check if "Decreases magical resistances of target" (spell penetration) works as intended because of this.

**Corruption Ticks (Warlock) (/w no spell penetration) versus target with 85 shadow resistance.** 

100% Damage Tick = 65 (amount / total amount = 65 / 69 = 94%)
75% Damage Tick = 2 (3%)
50% Damage Tick = 2 (3%)
25% Damage Tick = 0 (0%)
0% Damage Tick = 0 (0%)

**Corruption Ticks (Warlock) (/w 35 spell penetration) versus target with 85 shadow resistance.**

100% Damage Tick = 66 (amount / total amount = 66 / 69 = 96%)
75% Damage Tick = 3 (4%)
50% Damage Tick = 0 (3%)
25% Damage Tick = 0 (0%)
0% Damage Tick = 0 (0%)

**Expected behavior**: [Overview [4] damage reduction at enemy resistance level](http://i.imgur.com/KiY7jHd.png)

_Same method was used on the current (old, non-ptr, live) server_

**Corruption Ticks (Warlock) (/w no spell penetration) versus target with 85 shadow resistance.** 

100% Damage Tick = 37 (36%)
75% Damage Tick = 42 (41%)
50% Damage Tick = 20 (19%)
25% Damage Tick = 4 (4%)
0% Damage Tick = 0 (0%)

**Corruption Ticks (Warlock) (/w 35 spell penetration) versus target with 85 shadow resistance.**

100% Damage Tick = 67 (62%)
75% Damage Tick = 32 (30%)
50% Damage Tick = 7 (6%)
25% Damage Tick = 0 (0%)
0% Damage Tick = 2 (2%)

**Corruption Ticks (Warlock) (/w no spell penetration) versus target with 0 shadow resistance.**

100% Damage Tick = 42 (100%)
75% Damage Tick = 0 (0%)
50% Damage Tick = 0 (0%)
25% Damage Tick = 0 (0%)
0% Damage Tick = 0 (0%)

**Steps to reproduce the problem**:

1. Cast Corruption on target with magic resistance.
2. Count the damage done by ticks in the combat log. (see method above)

TL;DR: If it ain't broke, don't fix it.

**Source:** 

[1] [Spell Penetration and Resistances Explanation Webarchive](https://web.archive.org/web/20161015101615/https://dwarfpriest.wordpress.com/2008/01/07/spell-hit-spell-penetration-and-resistances/)

[2] [Reddit post on resistances by a cMangos developer](https://www.reddit.com/r/wowservers/comments/59tvha/cmangos_presents_first_fully_blizzlike_magic/)

[3] [cMangos Github discussion on resistances in classic/TBC](https://github.com/cmangos/issues/issues/1193)

[4] [Quick overview of resistances](http://vanillagaming.org/forum/index.php?topic=15951.0)

**Comment by MrKelzo** [comment]
Working as intended.
